kurl_util/cmd/pvmigrate/main.go

```diff
@@ -1,15 +1,124 @@
 package main
 
 import (
+	"context"
+	"flag"
 	"fmt"
+	"log"
+	"os"
+	"os/signal"
 
+	clusterspace "github.com/replicatedhq/kurl/pkg/cluster/space"
 	"github.com/replicatedhq/kurl/pkg/version"
 	"github.com/replicatedhq/pvmigrate/pkg/migrate"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
 )
 
 func main() {
```

Comment:
Should that to be a standalone command or subcommand of kurl?
When/who will call this command?